spec/service/form_task_list_service_spec.rb

```diff
@@ -422,6 +424,46 @@
               expect(section_rows.first[:task_name]).to eq "Make your form live"
               expect(section_rows.first[:path]).to eq "/forms/1/make-live"
             end
+
+            context "when form is ready to make live" do
```

Comment:
These changes look good, but I think go beyond what I'd call a refactor... does the commit message need to be updated?

Reply:
Good point, I don't think I properly considered the commit message. I've changed it to "Add missing tests for FormTaskListService when groups are enabled"